content/cloud-computing/concepts/saas/saas.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: 'SaaS'
+Description: 'SaaS is a cloud computing service delivery model that makes software and applications available over the internet.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cloud'
+  - 'Infrastructure'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'back-end-engineer-career-path'
+---
+
+Software as a service or **SaaS** is a cloud computing service delivery model that makes software and applications available over the internet.
+
+Some examples of SaaS include Google Workspace, Dropbox, and Microsoft 365.
+
+## Architecture
+
+SaaS services typically use a multi-tenant cloud architecture. This allows consumers to share computing resources within a cloud.
+
+Each tenant has their own space to store data. The data is isolated and will remain invisible to other tenants.
```

Comment:
```suggestion
SaaS services typically use a multi-tenant approach, where multiple consumers share access to cloud resources. Each tenant has their own space to store data. The data is isolated and will remain invisible to other tenants.
```

Reply:
@nikstern @KTom101 In the context of this entry, is "tenant" synonymous with "user" and "consumer"? And if so, should we pick one and go with that?

Reply:
@nikstern @Dusch4593 Not sure what would be preferred. When writing this I was going back and forth with the terminology. I'm thinking now that maybe "customer" may be more suitable (from a service provider's perspective). Let me know your thoughts.